internal/router/controllers/user.go

```diff
@@ -234,8 +236,9 @@ func (c *UserController) GetTableQuery(rw http.ResponseWriter, r *http.Request)
 		return
 	}
 
-	rw.WriteHeader(http.StatusOK)
+	c.collectReadQueryMetric(r, stm, config)
```

Comment:
Here's the first point that the read query metric is collected. The second point is [in the JSON RPC service](https://github.com/tablelandnetwork/go-tableland/pull/385/files#diff-62ca30c3e73e3613d5dacf5ca84bd0f83f386a0a2ae949142dc36322483bcc92R212).

Some observations:
- Only successful queries are being collected. I thought that collecting unsuccessful queries could be a bit spammy.
- I'm also collecting the format options. That way we can analyze if people are using those options
- The IP address is also being collected. It is captured by the [logging middleware](https://github.com/tablelandnetwork/go-tableland/pull/385/files#diff-8874654c7b1682c125a6e8dada089eb7b278ba1130a3687563b0f0a0baccbc49R13-R20) and passed via context.

Reply:
Realizing now that maybe we could capture the time it took to run? wdyt?

Reply:
> Realizing now that maybe we could capture the time it took to run? wdyt?

Yes, that sounds very useful.